core/jobs/batch_jobs/exp_proto_size_in_byte_addition_jobs.py

```diff
@@ -0,0 +1,74 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Jobs for adding proto_size_in_bytes attribute to exploration."""
+
+from __future__ import annotations
+
+from core.domain import exp_domain
+from core.jobs import base_jobs
+from core.jobs import job_utils
+from core.jobs.io import ndb_io
+from core.jobs.types import job_run_result
+from core.platform import models
+
+import apache_beam as beam
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import exp_models
+
+(exp_models,) = models.Registry.import_models([models.NAMES.exploration])
+
+datastore_services = models.Registry.import_datastore_services()
+
+
+class AddProtoSizeInBytesToExplorationJob(base_jobs.JobBase):
```

Comment:
@vojtechjelinek Could you please provide some initial feedback on this beam job? 
Please ignore the formatting errors. 

Steps I am trying to do here:
1. Get all the Exploration model
2. Migrate to version 55
3. Write back to datastore

Reply:
<img width="1423" alt="Screenshot 2021-12-15 at 10 06 18" src="https://user-images.githubusercontent.com/25057618/146124032-5c5bad43-6d9f-4a99-acf5-25189771680a.png">

Reply:
Also, to confirm, is it ok If I run this job locally, does this affect any important data or part of codebase? 
want to make sure and prevent any risk points.

<img width="860" alt="Screenshot 2021-12-15 at 10 08 00" src="https://user-images.githubusercontent.com/25057618/146124177-e80cff55-2f38-4888-859f-38012f0a7104.png">

Reply:
<img width="1422" alt="Screenshot 2021-12-15 at 10 42 05" src="https://user-images.githubusercontent.com/25057618/146128688-869488ff-520a-4c01-ad8b-cbcf64e7e90a.png">


But not able to check the daatstore viewer, not able to find ExplorationModel.

Reply:
Ah, I think it would be better to write a generic exploration migration job. I have started with similar jobs in this PR #14361

Yes, you can run the job locally. You can inspect the local datastore using https://github.com/remko/dsadmin

Reply:
Tried to write similar job, but getting error saying 
```
Input to Flatten must be an iterable. Got a value of type <class 'apache_beam.pvalue.PCollection'> instead.
```

Could you please help me here?

Reply:
The latest error I am facing now is:
```
google.cloud.ndb.exceptions.ContextError: No current context. NDB calls must be made in context established by google.cloud.ndb.Client.context. [while running 'Generate exploration summary models to put']
```

Will try to fix it tomorrow.

Reply:
CLosing this conversation as the job is passing now.